modules/fun/owoCord.js

```diff
@@ -1,13 +1,17 @@
-let version = "0.2.0";
+let version = "0.3.1";
 
 let owointerval;
 function owoify() {
+	String.prototype.owocordToOwOSpeak = function() {
```

Comment:
Could you not pollute JS prototypes like this and just use a normal function please? Thanks.

Reply:
Alright

Reply:
done